src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstanceDefinition.scala

```diff
@@ -236,33 +236,11 @@ object InstanceDefinition {
       t: TypePropertyDefinition
   ): Either[DecodingFailure, InstancePropertyValue] =
     t.`type` match {
-      case _: PropertyType.DirectNodeRelationProperty =>
+      case PropertyType.DirectNodeRelationProperty(_, _, Some(false)) |
+          PropertyType.DirectNodeRelationProperty(_, _, None) =>
```

Comment:
a little annoying that we have list check here, but also `t.isList` case later on, also why toInstancePropertyTypeOfNonList handles some non-lists, but DirectNodeRelationProperty is handled here

But that's how it was setup before, so making it cleaner isn't necessarily in scope for this PR

Reply:
You are right, no reasons to keep it there, as far as I can see